### PR TITLE
Quick soa fix

### PR DESCRIPTION
**Pull request by lfierce2**
So... I'm not sure how this is supposed to work. I assume someone other than me should review changes before it's merged? (I've merged a couple branches on my own, but these were minor bug fixes.)

### CODE REVIEW

**Comment by jeff-cohere** [comment]
I think this is how it's supposed to work! Let's see what tests are failing.

**Comment by jeff-cohere** [comment]
Here it is (you can click on the "Details" link next to the failing tests with the red X to see it yourself):

```
======================================================================
ERROR: test_create_inputs (test.test_mam4.TestMAM4AerosolModel.test_create_inputs)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/runner/work/ambrs/ambrs/test/test_mam4.py", line 233, in test_create_inputs
    inputs = model.create_inputs(self.ensemble, dt, nstep)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/ambrs/ambrs/ambrs/aerosol_model.py", line 74, in create_inputs
    inputs.append(self.create_input(scenario, dt, nstep))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/ambrs/ambrs/ambrs/mam4.py", line 190, in create_input
    aero_mass_fracs = AerosolMassFractions(scenario)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/ambrs/ambrs/ambrs/mam4.py", line 122, in __init__
    SOA = scenario.size.modes[0].mass_fraction('MSA'), # FIXME: using MSA as a placeholder for SOA
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/ambrs/ambrs/ambrs/aerosol.py", line 94, in mass_fraction
    index = [s.name for s in self.species].index(species_name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: 'MSA' is not in list
```

**Comment by jeff-cohere** [comment]
Wow, somehow the Git repository for this project has increased in size by ~1.3Gb. I'm not sure what's happened, but we might want to try to reduce it a bit for ease of collaboration. In the meantime, I'll poke at this error and see if I can figure out what's going on.

File: ambrs/aerosol.py [hunk 1]
```diff
@@ -89,6 +90,7 @@ class AerosolModeState:
     def mass_fraction(self, species_name: str):
         """returns the mass fraction corresponding to the given species name,
 or throws a ValueError."""
+        print(self.species)
```
Comment (jeff-cohere) [inline ambrs/aerosol.py:93]: We can probably get rid of this `print`.

File: ambrs/ppe.py [hunk 2]
```diff
@@ -198,10 +198,20 @@ def lhs(specification: EnsembleSpecification,
...
                 ) for m, mode in enumerate(specification.size.modes)]),
         )
-        # normalize mass fractions
+
+        # FIXME: laura's janky fix to set some mass fractions to zero
+        for mode in size.modes:
+            for qq in range(len(mode.mass_fractions)):
+                print(mode.mass_fractions[qq])
+                for ii in range(len(mode.mass_fractions[qq])):
+                    if np.isnan(mode.mass_fractions[qq][ii]):
```
Comment (jeff-cohere) [inline ambrs/ppe.py:207]: Are we seeing NaNs in mass fractions after creating an ensemble?

**Comment by lfierce2** [comment]
> Wow, somehow the Git repository for this project has increased in size by ~1.3Gb. I'm not sure what's happened, but we might want to try to reduce it a bit for ease of collaboration. In the meantime, I'll poke at this error and see if I can figure out what's going on.

**Review by lfierce2 — COMMENTED** [review]
I just accidentally closed a comment. I think I accidentally included mam4_runs and partmc_runs in one of the commits. I tried to remove them, but I don 't think it worked. I'll see what I can do!

File: ambrs/aerosol.py (hunk 1 above)
Comment (lfierce2) [reply ambrs/aerosol.py:93]: Ah yes. I was debugging. Thanks!

File: ambrs/ppe.py (hunk 2 above)
Comment (lfierce2) [reply ambrs/ppe.py:207]: NaNs started popping up when I attempted to zero-out some species. As a quick fix, I sampled from the uniform distribution from (0,0), which created NaNs. These led to all species in that mode going to NaNs after I normalized.

I think the best solution would be to implement an approach to set values to a fixed value across ensembles. This a feature we'd like for other reasons (e.g., running an ensemble all at the same RH, etc.). I'm sure there's an easier way to do this than sample a uniform distribution.

**Comment by lfierce2** [comment]
I accidentally closed this pull request (I think?), so I reopened it.

**Pull request merged**